Instagram/View Controllers/HomeViewController.m

```diff
@@ -113,6 +119,14 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         UINavigationController *navigationController = [segue destinationViewController];
         DetailsViewController *detailsVC = (DetailsViewController*)navigationController.topViewController;
         detailsVC.post = self.posts[myIndexPath.row];
+    } else if ([segue.identifier isEqual:@"profileSegue"]) {
```

Comment:
Because you use the string `@"profileSegue"` in multiple places in the code, you should consider making it a constant. Otherwise it's easy to change one spot but miss another.